reference/commands/list.rst

```diff
@@ -346,3 +346,38 @@ with the following structure:
       }
     }
   }
+
+List html output
+----------------
+
+The ``conan list ... --format=html`` will return a html output in ``stdout`` (can be redirected to file)
+with the following structure:
+
+.. code-block:: text
+
+  $ conan list "zlib/1.2.12#*:*" --format=html
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <title>conan list results</title>
+      <script>
+        var list_results = {"Local Cache": {"zlib/1.2.12": {"revisions": {"c2cd5ec484e443e4f01b51ff5370fd11": {"timestamp": 1665747741.12, "packages": {"6fe7fa69f760aee504e0be85c12b2327c716f9e7": {"info": {"settings": {"arch": "x86_64", "build_type": "Release", "compiler": "apple-clang", "compiler.version": "14", "os": "Macos"}, "options": {"fPIC": "True", "shared": "False"}}}}}, "0de8ff7f99079cd07341311c9ead89a2": {"timestamp": 1676913747.767964, "packages": {"6fe7fa69f760aee504e0be85c12b2327c716f9e7": {"info": {"settings": {"arch": "x86_64", "build_type": "Release", "compiler": "apple-clang", "compiler.version": "14", "os": "Macos"}, "options": {"fPIC": "True", "shared": "False"}}}}}}}}};
```

Comment:
I think the specifics of the output file are irrelevant. I'd document that you can open it, or maybe do a screen capture (or embed the output html) to display the actual result